src/pages/workspaces/workspace/analysis/modals/CloudEnvironmentModal.js

```diff
@@ -378,27 +381,30 @@ export const CloudEnvironmentModal = ({
     ])
   }
 
-  const NEW_JUPYTER_MODE = tools.Jupyter.label
-  const NEW_RSTUDIO_MODE = tools.RStudio.label
-  const NEW_GALAXY_MODE = tools.Galaxy.label
-  const NEW_CROMWELL_MODE = tools.Cromwell.label
-  const NEW_AZURE_MODE = tools.Azure.label
+  const getGCPView = () => Utils.switchCase(viewMode,
+    [toolLabels.Jupyter, () => renderComputeModal(toolLabels.Jupyter)],
+    [toolLabels.RStudio, () => renderComputeModal(toolLabels.RStudio)],
+    [toolLabels.Galaxy, () => renderAppModal(GalaxyModalBase, toolLabels.Galaxy)],
+    [toolLabels.Cromwell, () => renderAppModal(CromwellModalBase, toolLabels.Cromwell)],
+    [Utils.DEFAULT, renderDefaultPage]
+  )
 
-  const getView = () => Utils.switchCase(viewMode,
-    [NEW_JUPYTER_MODE, () => renderComputeModal(NEW_JUPYTER_MODE)],
-    [NEW_AZURE_MODE, () => renderAzureModal()],
-    [NEW_RSTUDIO_MODE, () => renderComputeModal(NEW_RSTUDIO_MODE)],
-    [NEW_GALAXY_MODE, () => renderAppModal(GalaxyModalBase, NEW_GALAXY_MODE)],
-    [NEW_CROMWELL_MODE, () => renderAppModal(CromwellModalBase, NEW_CROMWELL_MODE)],
+  const getAzureView = () => Utils.switchCase(viewMode,
+    [toolLabels.JupyterLab, renderAzureModal()],
```

Comment:
`Utils.switchCase` expects a function to call. Think it got lost in the refactor, you're calling the function early here. This causes the Azure modal to break.

Reply:
> `Utils.switchCase` expects a function to call. Think it got lost in the refactor, you're calling the function early here. This causes the Azure modal to break.